Fortinet/fortigate/_meta/fields.yml

```diff
@@ -0,0 +1,29 @@
+rule.apprisk:
+  name: rule.apprisk
+  type: keyword
+  description: ""
+
+dns.rrname:
+  name: dns.rrname
+  type: keyword
+  description: ""
+
+tmp.icmp_code:
+  name: tmp.icmp_code
+  type: keyword
+  description: ""
+
+tmp.icmp_type:
+  name: tmp.icmp_type
+  type: keyword
+  description: ""
+
+fortigate.event.severity:
+  name: fortigate.event.severity
+  type: keyword
+  description: ""
+
+dns.rrtype:
```

Comment:
You can use [dns.question.type](https://www.elastic.co/guide/en/ecs/current/ecs-dns.html#field-dns-question-type)